### ISSUE TITLE
Reveal context like a path in the minibuffer or copy that info to the kill ring

### ISSUE THREAD

**Issue by techapu** [Kungsgeten/org-brain#166]
Hi! When I'm on org-mode I can use `org-display-outline-path` to display the complete path to the headline in the minibuffer. In this way, one can capture the context in a glance.
When I'm on org-brain, one can hide or show ascendents or descendants with + or -. But when I look at the friends, or the pinned entries, I can't figure out what parent or grandparent is, without leaving the entry I am in. And sometimes I have multiple entries with similar names and diferent paths. 
That would be easy if I could call a function at point that "reveal" path context of the friend or pinned entry, or any entry, in the minibuffer (like eww or w3m does with the url info). Added to that, it would be a great help if one could kill that path to the kill ring for working reference, or even use that info as an alternative "url" info for that thougth. 
And that would be helpful too when having multiple brains, and using the BRAINSWITCH facilities. In that case, it would be practical to be able to retrieve the "relative" path (inside the working brain), or the "absolute" path (the file first, the headline path after, sort of way).
Thanks!

**Comment by Kungsgeten** [comment]
Sorry, I don't quite understand what you want to do. Am I correct in that you'd like two features:

1. Being able to "visualize" a link in `org-brain-visualize`, without exiting the current "visualize session". What this means is that there could be several `*org-brain*` buffers at the same time.
2. If on a link in `org-brain-visualize` you'd like to store that link using `org-store-link` or similar?

**Comment by techapu** [comment]
No, that's not, I'm sorry. It's a "simple" feature what I am suggesting. I'll try to put it what you enumerate as (1) in an example.
First of all, I'm not talking of a link like in a "resource" link. I'm focusing in the outline path of the "real" org-mode headline behind the org-brain entry. And it's for closer integration with the org-mode workflow, so it's more intuitive with current existing functions in org-mode.
And it is not for the "current thought", it's suggested for the other entries, mainly for the "friends" and the "pinned" entries. That entries one can see, but can't grasp the whole context without visiting the entry entirely leaving the actual one behind. The user loses focus, or couldn't know what that entry is about from the actual view.  

Now, to achieve what I look for, I have to make a few steps (that distracts the user from the objective). Suppose I'm over a thought in org-brain buffer: I have to open the thought via `org-brain-goto-current` function, then, in the org-mode buffer, I have to call `org-display-outline-path` (try that function on an org-mode headline to follow me), so I can view all the way to the first parent: the "path" of the outline displayed in the mini-buffer. 
But if you code it, I think you can accomplish some "easy" function than returns the same output to the mini-buffer without actually visiting the file, nor even leave the org-brain buffer. It's a little more clear for the mind, while you are working. 
An abstract sintax for the outline path I'm talking about would be some near this:

--------------
Mode-line
----------------------------------------
Mini-buffer output: "foo.org/headline1/headline2/headline3/CurrentEntry"

You have to recognize that the org-brain interface is, even if sofisticated, very minimal, so one can see the title of the org-brain entry but it's not clear the whole map, only the nearer context in the mindmap view. 

Other Emacs packages have more customizable views, like dired, where you can see a detailed view, or a barebones one, customizing the interface.
Thanks!

**Comment by techapu** [comment]
In a response to the point you enumerate as (2), I wasn't talking about links but about org-brain entries. 
So I was also suggesting that a good way to identify an org-brain entry would be not only with an ID but also with the outline path, like it could be done in org-mode. 
And that this info outputs from an explicit org-brain function that returns that To output the path from an entry as a result. 
I think any of this suggestions need "several org-brain buffers at the same time".

**Comment by Kungsgeten** [comment]
OK, I think I understand what you're asking for. It sounds useful and not that hard to implement :)

**Comment by Kungsgeten** [comment]
Hi! Could you try this and see if it does what you want? It should work if you place the point over an entry link when visualizing. If you don't have it over a link, it will use the visualized entry.

```
(defun org-brain-outline-path-at-point ()
  (interactive)
  (let ((entry (or (ignore-errors (car (org-brain-button-at-point)))
                   (org-brain-entry-at-pt))))
    (if (org-brain-filep entry)
        (message "%s.%s" entry org-brain-files-extension)
      (org-with-point-at (org-brain-entry-marker entry)
        (org-display-outline-path t t)))))
```

**Comment by Kungsgeten** [comment]
@techapu Have you had the time to look at the function?
